src/test/java/seedu/address/logic/commands/client/AddClientCommandTest.java

```diff
@@ -249,6 +254,16 @@ public void sortSession() {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public List<Schedule> findScheduleBySession(Session sessionKey) {
```

Comment:
Just wondering why these few added methods here return null

Reply:
thanks for spotting them, that was auto-filled by IntelliJ (because ModelStub implements Model)
will change it